yt_dlp/extractor/vrt.py

```diff
@@ -1,82 +1,216 @@
-from .common import InfoExtractor
+import json
+
+from time import (strftime, gmtime)
+from .gigya import GigyaBaseIE
+from ..compat import compat_HTTPError
 from ..utils import (
-    extract_attributes,
+    parse_iso8601,
+    ExtractorError,
+    int_or_none,
     float_or_none,
-    get_element_by_class,
-    strip_or_none,
-    unified_timestamp,
+    str_or_none,
+    url_or_none,
+    urlencode_postdata
 )
 
 
-class VRTIE(InfoExtractor):
-    IE_DESC = 'VRT NWS, Flanders News, Flandern Info and Sporza'
-    _VALID_URL = r'https?://(?:www\.)?(?P<site>vrt\.be/vrtnws|sporza\.be)/[a-z]{2}/\d{4}/\d{2}/\d{2}/(?P<id>[^/?&#]+)'
+class VRTIE(GigyaBaseIE):
+    IE_DESC = 'VRT'
+    _VALID_URL = r'https?://(?:www\.)?vrt\.be/vrtnu/a-z/(?:[^/]+/){2}(?P<id>[^/?#&]+)'
     _TESTS = [{
-        'url': 'https://www.vrt.be/vrtnws/nl/2019/05/15/beelden-van-binnenkant-notre-dame-een-maand-na-de-brand/',
-        'md5': 'e1663accf5cf13f375f3cd0d10476669',
-        'info_dict': {
-            'id': 'pbs-pub-7855fc7b-1448-49bc-b073-316cb60caa71$vid-2ca50305-c38a-4762-9890-65cbd098b7bd',
-            'ext': 'mp4',
-            'title': 'Beelden van binnenkant Notre-Dame, één maand na de brand',
-            'description': 'Op maandagavond 15 april ging een deel van het dakgebinte van de Parijse kathedraal in vlammen op.',
-            'timestamp': 1557924660,
-            'upload_date': '20190515',
-            'duration': 31.2,
-        },
-    }, {
-        'url': 'https://sporza.be/nl/2019/05/15/de-belgian-cats-zijn-klaar-voor-het-ek/',
-        'md5': '910bba927566e9ab992278f647eb4b75',
+        'url': 'https://www.vrt.be/vrtnu/a-z/heizel-1985/trailer/heizel-1985-trailer/',
         'info_dict': {
-            'id': 'pbs-pub-f2c86a46-8138-413a-a4b9-a0015a16ce2c$vid-1f112b31-e58e-4379-908d-aca6d80f8818',
+            'id': 'pbs-pub-e1d6e4ec-cbf4-451e-9e87-d835bb65cd28$vid-2ad45eb6-9bc8-40d4-ad72-5f25c0f59d75',
+            'title': 'Trailer \'Heizel 1985\'',
+            'thumbnail': 'https://images.vrt.be/orig/2022/09/07/6e44ce6f-2eb3-11ed-b07d-02b7b76bf47f.jpg',
             'ext': 'mp4',
-            'title': 'De Belgian Cats zijn klaar voor het EK mét Ann Wauters',
-            'timestamp': 1557923760,
-            'upload_date': '20190515',
-            'duration': 115.17,
+            'duration': 41.05,
+            'release_date': '20220908',
+            'description': 'md5:816dcd9e3be706b16e4e32e3f723a5a1',
```

Comment:
This seems to have changed:

```suggestion
            'description': 'md5:48f8b2cddc819ddb2d21455be1ace6ef',
```